www/media/css/language.css

```diff
@@ -0,0 +1,24 @@
+.flag {
+	width: 16px !important;
+	height: 10px !important;
```

Comment:
I had to use `!important` here since the bootstrap selector thingy reduces the flag images and without proper CSS skills, this was the only solution I found.

Any CSS guy/gal that could help me out here?